VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoCosmosSwitch.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  TransactionMemoCosmosIBC.swift
+//  VultisigApp
+//
+//  Created by Enrique Souza Soares on 17/05/24.
+//
+
+import SwiftUI
+import Foundation
+import Combine
+
+/**
+
+ 2) COSMOS - FUNCTION: “SWITCH THORCHAIN”
+
+ UI Elements:
+     •    Address Field:
+     •    Prefilled with the user’s THORChain address
+     •    Allow manual override
+     •    Amount Field: Enter amount to switch
+
+ Action:
+ → Send MsgSend from COSMOS to THORChain vault
+ → Include memo: SWITCH:<thorAddress>
+ 
+ */
+
+class TransactionMemoCosmosSwitch: TransactionMemoAddressable, ObservableObject {
+    @Published var amount: Double = 0.0
+    @Published var destinationAddress: String = "cosmos144za3huzmfl3k6hge487a6gnu06vwk6t2hfk53"
```

Comment:
I think this destination address should be read from `https://thornode.ninerealms.com//thorchain/inbound_addresses` 
we should never hardcode the inbound address , as THORChain will churn every three days, if we hard coded it , user will lose fund